packages/sdk-core/src/modules/converter/StackTraceConverter.ts

```diff
@@ -0,0 +1,161 @@
+import { BacktraceStackFrame } from '../../model/data/BacktraceStackTrace';
+import { JavaScriptEngine } from '../../model/data/JavaScriptEngine';
+
+export class StackTraceConverter {
+    public readonly UNKNOWN_FRAME = 'unknown';
+    private readonly ANONYMOUS_FRAME: Record<JavaScriptEngine, string> = {
+        ['firefox']: 'debugger eval code',
+        ['v8']: '<anonymous>',
+        ['safari']: '[native code]',
```

Comment:
How did you get those values? I only get them in Firefox and Chrome when using the console, but when I serve a file, those are not printed. I've never seen `[native code]` of Safari even once.

### Firefox 
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/778a2819-e938-4ff4-8ae3-fc3d27103a6d)

### Chrome
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/b6abc847-c515-41fd-bf17-d8c8ffe0fd1a)

### Safari
![image](https://github.com/backtrace-labs/backtrace-javascript/assets/9085864/bc776b96-e408-4ce0-a427-37994142261a)


### Testing code

```
function lambdaAnonymousTest() {
    (() =>
        (() =>
            (() => {
                throw new Error('lambdaAnonymousTest');
            })())())();
}

function anonymousTest() {
    (function () {
        (function () {
            (function () {
                throw new Error('anonymousTest');
            })();
        })();
    })();
}

function namedLambdaAnonymousTest() {
    const throwing = () => {
        throw new Error('namedLambdaAnonymousTest');
    };
    const anon2 = () => throwing();
    const anon1 = () => anon2();
    anon1();
}

function namedTest() {
    function throwing() {
        throw new Error('namedTest');
    }

    function anon2() {
        throwing();
    }

    function anon1() {
        anon2();
    }

    anon1();
}

function test(fun) {
    try {
        fun();
    } catch (err) {
        console.log(err.message, err.stack);
    }
}

test(lambdaAnonymousTest);
test(anonymousTest);
test(namedLambdaAnonymousTest);
test(namedTest);
```

Reply:
done.